_format/1.0/index.md

```diff
@@ -240,13 +240,14 @@ alongside other information to be represented in a resource object, these keys
 #### <a href="#document-resource-object-relationships" id="document-resource-object-relationships" class="headerlink"></a> Relationships
 
 The value of the `relationships` key **MUST** be an object (a "relationships
-object"). Members of the relationships object ("relationships") represent
-references from the [resource object][resource objects] in which it's defined to other resource
-objects.
+object"). Members of the relationships object represent "relationships", i.e.,
+references from the [resource object][resource objects] in which it's defined
+to other resource objects.
 
 Relationships may be to-one or to-many.
 
-A "relationship object" **MUST** contain at least one of the following:
```

Comment:
This needs to be updated in a few more places:
https://github.com/json-api/json-api/blob/gh-pages/_format/1.0/normative-statements.json:
https://github.com/json-api/json-api/blob/gh-pages/_format/1.1/index.md
https://github.com/json-api/json-api/blob/gh-pages/_format/1.1/normative-statements.json: